libminifi/src/core/ProcessGroup.cpp

```diff
@@ -133,6 +133,7 @@ void ProcessGroup::startProcessingProcessors(TimerDrivenSchedulingAgent& timeSch
   for (const auto processor : failed_processors_) {
     try {
       logger_->log_debug("Starting %s", processor->getName());
+      processor->setScheduledState(core::ScheduledState::RUNNING);
```

Comment:
Scheduling state needs to be reset to RUNNING if FlowController is stopped through controller or C2 otherwise scheduler will not be able to restart the processor.